spec/services/shops_list_service_spec.rb

```diff
@@ -38,5 +47,14 @@
       expect(shop.association(:logo_attachment).loaded?).to be true
       expect(shop.logo.association(:blob).loaded?).to be true
     end
+
+    it "fetches enterprises without active order cycles" do
+      open_enterprise_ids = Enterprise.distributors_with_active_order_cycles.pluck(:id).to_set
+      expect(open_enterprise_ids).not_to be_empty
+
+      shops.pluck(:id).each do |shop_id|
```

Comment:
Here I would find it much clearer to state something like `expect(shops).to eq distributors[1..2]`.